styleguide/source/assets/scss/02-molecules/_event-stub.scss

```diff
@@ -54,7 +69,9 @@
 
   .ama__image {
     width: 100%;
+    min-width: 180px;
     height: auto;
+    @include gutter($margin-bottom-full...);
```

Comment:
All `@includes` need to be the first value

@include gutter($margin-bottom-full...);
width: 100%;
min-width: 180px;
height: auto;